source/d3d11/draw_call_tracker.cpp

```diff
@@ -59,21 +75,35 @@ namespace reshade::d3d11
 		}
 	}
 
-	ID3D11DepthStencilView* draw_call_tracker::get_best_depth_stencil(UINT width, UINT height)
+	ID3D11DepthStencilView* draw_call_tracker::get_best_depth_stencil(std::string host_process_name, std::unordered_map<std::string, runtime::game> game_list, UINT drawcalls, const com_ptr<ID3D11Device> device, const com_ptr<ID3D11DeviceContext> devicecontext, UINT width, UINT height)
 	{
 		depthstencil_counter_info best_info = { 0 };
 		ID3D11DepthStencilView *best_match = nullptr;
 		float aspect_ratio = ((float)width) / ((float)height);
+		static runtime::game current_game =
```

Comment:
This isn't used, so might as well remove it.

Reply:
OK. I made it because on my first attempts, i added hacks per game in the post processing stage, in the draw_call_tracker get_best_depthstencil method.